src/main/java/com/webtutsplus/ecommerce/controller/OrderController.java

```diff
@@ -0,0 +1,140 @@
+package com.webtutsplus.ecommerce.controller;
+
+import com.stripe.Stripe;
+import com.stripe.exception.StripeException;
+import com.stripe.model.checkout.Session;
+import com.stripe.param.SubscriptionCreateParams;
+import com.stripe.param.checkout.SessionCreateParams;
+import com.webtutsplus.ecommerce.common.ApiResponse;
+import com.webtutsplus.ecommerce.dto.ProductDTOs.AddToCartDto;
+import com.webtutsplus.ecommerce.dto.ProductDTOs.CartDto;
+import com.webtutsplus.ecommerce.dto.ProductDTOs.OrderDto;
+import com.webtutsplus.ecommerce.dto.ProductDTOs.PlaceOrderDto;
+import com.webtutsplus.ecommerce.exceptions.AuthenticationFailException;
+import com.webtutsplus.ecommerce.exceptions.ProductNotExistException;
+import com.webtutsplus.ecommerce.model.*;
+import com.webtutsplus.ecommerce.service.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.*;
+
+@RestController
+@RequestMapping("/order")
+public class OrderController {
+    @Autowired
+    private OrderService orderService;
+
+    @Autowired
+    private CartService cartService;
+
+    @Autowired
+    private AuthenticationService authenticationService;
+
+    @Autowired
+    OrderItemsService orderItemsService;
+
+    private String sessionId;
+
+    @PostMapping("/add")
+    public ResponseEntity<ApiResponse> addOrder(@RequestParam("token") String token,@RequestParam("sessionId") String sessionId) throws ProductNotExistException {
```

Comment:
make throws ProductNotExistException,  AuthenticationFailException and remove try catch